File: jdroid-java-webapp-core/src/test/java/com/jdroid/javaweb/config/ConfigHelperTest.java
package com.jdroid.javaweb.config;

import com.jdroid.javaweb.application.Application;
import com.jdroid.javaweb.application.TestBuildConfig;
import com.jdroid.javaweb.commons.AbstractTest;

import org.testng.Assert;
import org.testng.annotations.Test;

public class ConfigHelperTest extends AbstractTest {
	
	@Test
	public void test() {
		Assert.assertEquals(Application.get().getRemoteConfigLoader().getString(CoreConfigParameter.APP_NAME), TestBuildConfig.APP_NAME);
	}
}

File: jdroid-java-webapp-core/src/test/java/com/jdroid/javaweb/config/ConfigHelperTest.kt
package com.jdroid.javaweb.config

import com.jdroid.javaweb.application.Application
import com.jdroid.javaweb.application.TestBuildConfig
import com.jdroid.javaweb.commons.AbstractTest
import org.junit.Assert
import org.junit.Test

class ConfigHelperTest : AbstractTest() {

    @Test
    fun test() {
        Assert.assertEquals(Application.get().remoteConfigLoader.getString(CoreConfigParameter.APP_NAME), TestBuildConfig.APP_NAME)
    }
}
